Bottom: 495792950a1021fa6f12ae3b326d20acfe54ec3c
Top:    4019690013ee59e234a7b014331f152e47ecc42f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 15:59:35 +0100

Refresh of provide-a-progress-dialog-for-

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 88501d9..1178292 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -92,6 +92,7 @@ RCSID ("$Id$");
 #define ROUTE_VERBOSE
 */
 
+/*
 #define ROUTE_DEBUG
 //#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
@@ -100,6 +101,7 @@ RCSID ("$Id$");
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
 //#define DEBUG_SHOW_ZIGZAG
+*/
 
 static direction_t
 directionIncrement(direction_t dir)
